packages/connection/src/class-manager.php

```diff
@@ -719,6 +719,21 @@ public function api_url( $relative_url ) {
 		$api_base = $api_base ? $api_base : 'https://jetpack.wordpress.com/jetpack.';
 		$version  = '/' . Utils::get_jetpack_api_version() . '/';
 
+		/**
+		 * Filters whether the connection manager should use the iframe authorization
+		 * flow instead of the regular redirect-based flow.
+		 *
+		 * @since 8.3.0
+		 *
+		 * @param Boolean $is_iframe_flow_used should the iframe flow be used, defaults to true.
+		 */
+		$iframe_flow = apply_filters( 'jetpack_use_iframe_authorization_flow', true );
```

Comment:
After discussing in Slack, Kim and I agree it is best to default to the classic flow until the Jetpack plugin adopts the iframe connection agin as default.

Reply:
Thanks, I have pushed a change that flips the default state.